bucket/stremio.json

```
@@ -0,0 +1,23 @@
+{
+    "version": "4.4.142",
+    "description": "Stremio media center",
```

```suggestion
    "description": "A modern media center that's a one-stop solution for your video entertainment. You discover, watch and organize video content from easy to install addons.",
```
This description is taken from their homepage.
`"Stremio media center"` looks a bit confusing for users that are not familiar with it. As the text didn't explain what Stremio is.